_posts/2019-09-26-2019-building-admissions.md

```diff
@@ -0,0 +1,284 @@
+## Building Elixir School's Admissions application
+
+If you didn't know, Elixir School has its own Slack where contributors can gather to discuss our organization's content and projects but most importantly, support one another in our Elixir journey. When we set out to create our own Slack we wanted to address a big concern with many public Slacks: the signal to noise ratio is bad, there's just too much spam. 
+
+> Have you contributed to an Elixir School project but not joined us on Slack? Head over to https://admissions.elixirschool.com to get your invite today!
+
+So how can we keep our Slack public but prevent spammers from joining and do so in a way that doesn't add work to our maintainers? Our solution: required at least once contribution of any side to any one of our projects. 
```

Comment:
maybe `required at least one contribution to any one of our projects` sounds better?

Reply:
I would also change `required` -> `requiring` or say `we could require at least one...`